frontend/src/components/summary/group/RFISummary.vue

```diff
@@ -614,12 +610,25 @@ export default {
       }
     };
   },
+  methods: {
+    getValueString(val){
+      if (val === 1){
+        return 'YES';
+      }
+      else if (val === 0){
+        return 'NO';
+      }
+
+      return val;
+    }
+  },
+  mounted() {
```

Comment:
This mounted value was removed from main, can you update the code so that we are not modifying mounted and watch?